### PR TITLE
Added search in documentation.

### PR DESCRIPTION
**Pull request by novykh**

### CODE REVIEW

File: src/organisms/documentation/search/provider.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    },
+  },
+  alwaysSearchOnInitialLoad: false,
+}
+
+const SearchProvider = ({ children }) => {
+  return (
+    <ElasticSearchProvider config={config}>
+      <WithSearch
+        mapContextToProps={({ searchTerm, setSearchTerm, results, reset }) => ({
```
Comment (apostolidhs) [inline src/organisms/documentation/search/provider.js:41]: Could be defined in a constant function.

File: src/organisms/documentation/search/results.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+const SearchResults = ({ results }) => {
+  const tabbedResults = useTabbedResults(results)
+
+  return (
+    <Flex overflow={{ vertical: "auto" }} data-testid="searchResults" flex>
+      <StyledTabs>
+        {keys.map(key => {
+          const tabResults = tabbedResults[tabValuesByKey[key]]
+          if (!tabResults?.length) return
```
Comment (apostolidhs) [inline src/organisms/documentation/search/results.js:72]: This filtering could be moved out in a useMemo section, so we do not return empty elements during the rendering.

File: src/organisms/documentation/search/results.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+const Link = styled.a`
+  text-decoration: none;
+`
+
+const Section = ({ title, description, url }) => (
+  <Flex gap={2} column>
+    <Text strong>{title?.snippet || "Untitled"}</Text>
+    {!!description && <Text>{description?.snippet || "No description"}&hellip;</Text>}
+    <Flex alignSelf="end">
+      <Text as={Link} color="primary" href={url} target="_blank">
```
Comment (apostolidhs) [inline src/organisms/documentation/search/results.js:16]: If the `url={url?.raw}` can be falsy, we could handle it without providing an invalid link

**Review by apostolidhs — APPROVED** [review]

File: src/organisms/documentation/search/results.js (hunk 2 above)
Comment (novykh) [reply src/organisms/documentation/search/results.js:72]: You mean return the actual keys back that have values?

File: src/organisms/documentation/search/results.js (hunk 3 above)
Comment (novykh) [reply src/organisms/documentation/search/results.js:16]: it can't be I will remove the `?`

**Pull request merged**